ansible.cfg

```diff
@@ -9,3 +9,6 @@ connection_plugins = plugins/connection
 callback_plugins = plugins/callbacks
 transport = monkeypatched_ssh
```

Comment:
Yeah I saw that too, and was going to address that in a different PR.

This PR is mostly to see if things get any faster in Jenkins. They /feel/ faster from my laptop but I would like to see some real numbers.